lib/policyEvaluator/evaluator.js

```diff
@@ -133,11 +133,17 @@ function meetConditions(requestContext, statementCondition, log) {
             // reference_policies_multi-value-conditions.html)
             const keyBasedOnRequestContext =
                 findConditionKey(key, requestContext);
-            // Handle IfExists
+            // Handle IfExists and negation operators
+            const operatorsWithNegation = ['StringNotEquals',
```

Comment:
Is there a list of available operators somewhere? That would be great to have. Also, why construct the array every time instead of having it available globally once and for all?